src/datagrid/DataGridFilter.tsx

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+import {classNames} from "../utils";
+import {ClassNames} from "./ClassNames";
+import {Button} from "../forms/button";
+import {DataGridRow} from "./DataGrid";
+
+type DataGridFilterProps = {
+    style?: any;
```

Comment:
Please add classname to Props if applicable.

Reply:
done